source/ed/types.h

```diff
@@ -146,6 +146,7 @@ struct CommercialMode : public Header, Nameable{
 
 struct PhysicalMode : public Header, Nameable{
     const static nt::Type_e type = nt::Type_e::PhysicalMode;
+    double co2_emission;
```

Comment:
to be sure this field is correctly initialized by the default constructor, you can write this line
```c++
    double co2_emission = 0.;